pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -50,15 +52,10 @@ func GetBootstrapOpenshiftRoles(openshiftNamespace string) []authorizationapi.Ro
 				Namespace: openshiftNamespace,
 			},
 			Rules: []authorizationapi.PolicyRule{
-				{
-					Verbs:     sets.NewString("get", "list"),
-					Resources: sets.NewString("templates", authorizationapi.ImageGroupName),
-				},
-				{
-					// so anyone can pull from openshift/* image streams
-					Verbs:     sets.NewString("get"),
-					Resources: sets.NewString("imagestreams/layers"),
-				},
+				authorizationapi.NewRule(read...).Groups(templateGroup).Resources("templates").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(imageGroup).Resources("imagestreams", "imagestreammappings", "imagestreamtags", "imagestreamimages").RuleOrDie(),
```

Comment:
Seems like it: https://github.com/openshift/origin/blob/master/pkg/image/registry/imagestreammapping/rest.go